blocks/init/src/Blocks/components/video-button/video-button.php

```diff
@@ -37,5 +38,5 @@ class="<?php echo \esc_attr($videoButtonClass); ?>"
 	data-micromodal-trigger="<?php echo \esc_attr($videoButtonModalId); ?>"
 	aria-label="<?php echo \esc_html($videoButtonLabel); ?>"
 >
-	<?php echo \wp_kses_post($icon); ?>
+	<?php echo $manifest['icon']; ?>
```

Comment:
Ignore the PHPCS no kses error here.

Reply:
Should the SVG icons be escaped? In theory they can contain js scripts. But we are controlling them from the manifest so it's safe, right?